src/TunnelEncapsulation.cc

```diff
@@ -14,6 +14,7 @@ EncapsulatingConn::EncapsulatingConn(Connection* c, BifEnum::Tunnel::Type t)
       src_port(c->OrigPort()),
       dst_port(c->RespPort()),
       proto(c->ConnTransport()),
+      proto_id(c->KeyProto()),
```

Comment:
In the other places we named the field `ip_proto` when living alongside an existing `proto` field, so I'd suggest doing this here too.